activejob/test/jobs/retry_job.rb

```diff
@@ -18,6 +19,7 @@ class CustomDiscardableError < StandardError; end
 
 class RetryJob < ActiveJob::Base
   retry_on DefaultsError
+  retry_on DisabledJitterError, jitter: nil
```

Comment:
Should we have a test for `0.0` As well?

Reply:
It's probably prudent. I'll add.

Reply:
Added `test "retry jitter disabled with zero"` https://github.com/rails/rails/pull/38003/files#diff-a7e74ec284b847138992693bd53c4746R173